packages/pharos/src/components/table/pharos-table.scss

```diff
@@ -7,20 +7,38 @@
 
 .table {
   border-collapse: collapse;
+}
 
-  th {
-    border: 1px solid var(--pharos-table-header-color-border, var(--pharos-color-ui-30));
-    padding: var(--pharos-spacing-1-x);
-  }
+.table-header {
+  background-color: var(--pharos-table-header-color-background, var(--pharos-color-ui-20));
+}
+
+.table-header__cell {
+  border: 1px solid var(--pharos-table-header-color-border, var(--pharos-color-ui-30));
+  padding: var(--pharos-spacing-1-x);
+}
 
-  td {
-    border: 1px solid var(--pharos-table-body-color-border, var(--pharos-color-ui-30));
-    padding: var(--pharos-spacing-1-x);
+.table-header--has-sticky-header {
+  position: sticky;
+  top: 0;
+}
+
+.table-header--has-active-sticky-header {
+  box-shadow: var(--pharos-elevation-level-2);
+
+  .table-header__cell {
+    border: none;
+    border-right: 1px solid var(--pharos-table-header-color-border, var(--pharos-color-ui-30));
+
+    &:last-child {
+      border-right: none;
+    }
   }
```

Comment:
Typically with BEM I think this would be `.table-header__cell--has-active-sticky-header` maybe? Involves adding a bit more fiddling at the render level, I suppose.

Reply:
Yeah, I questioned that a bit too, no strong opinion really. I can try that out in the morning, we just need do a querySelectorAll for the `th` elements and loop over all of them to toggle a class with the IntersectionObserver flips.

I also wondered if we would be better served having separate `table-header` and `table-sticky-header` block level classes which would let the modifier be `--is-active` 

Having both `--has-sticky-header` and `--has-active-stick-header` feels weird to me, but I'm not particularly strong with BEM

Reply:
Okay, updated with block level classes for the sticky header, I updated the unit tests, I couldn't get the scrolling to work but it still has pretty good coverage